spec/requests/providers/client_received_legal_helps_spec.rb

```diff
@@ -0,0 +1,119 @@
+require 'rails_helper'
+
+RSpec.describe 'provider own home requests', type: :request do
```

Comment:
It looks like this was copied over from another spec 😉. The description should probably be changed.